src/includes/model/export/class-sg-export-cart.php

```diff
@@ -91,6 +91,7 @@ protected function set_external_coupons() {
 			$coupons = [];
 			foreach ( $this->shopgate_cart->getExternalCoupons() as $external_coupon ) {
 
+				$external_coupon->setCode(wc_format_coupon_code( $external_coupon->getCode()) );
```

Comment:
Could you please add a whitespace before wc_format_coupon_code() -> setCode( wc_format_coupon_code(.
WooCommerce style :)

Reply:
This has been corrected.